src/main/java/jp/seekengine/trainingjava/controller/ScheduleController.java

```diff
@@ -149,27 +161,39 @@ private String formatToISO8601(Duration duration) {
 
     @GetMapping("/end")
     public SampleTimeResponse calculateEndTime(@RequestBody SampleTimeRequest sampleTimeRequest) {
-        //StartTimeとDurationをインナーレコードにした形式
-        SampleTimeRequest.StartTime startTime = sampleTimeRequest.startTime();
-        SampleTimeRequest.Duration duration = sampleTimeRequest.duration();
-        //jp.seekengine.trainingjava.controller.request.Duration duration = sampleTimeRequest.duration();
-        ZonedDateTime startDateTime = ZonedDateTime.of(
-                startTime.year(),
-                startTime.month(),
-                startTime.date(),
-                startTime.hour(),
-                startTime.minute(),
-                startTime.second(),
-                0,
-                ZoneId.of("Asia/Tokyo")
-        );
-
-        ZonedDateTime endDateTime = startDateTime.plusHours(duration.hour())
-                .plusMinutes(duration.minute())
-                .plusSeconds(duration.second());
-
-
-        return new SampleTimeResponse(endDateTime.format(DateTimeFormatter.ISO_OFFSET_DATE_TIME));
+        try {
+            //StartTimeとDurationをインナーレコードにした形式
```

Comment:
直さなくても良いですが、通常コメントは//の後にスペースを入れることが多いので覚えておいてください。
```suggestion
            // StartTimeとDurationをインナーレコードにした形式
```